setup-env

```diff
@@ -39,6 +39,51 @@ python_versions() {
   pyenv versions --bare --skip-aliases --skip-envs
 }
 
+check_python_version() {
+  local version=$1
+
+  # This is a valid regex for semantically correct Python version strings.
+  # For more information see here: https://regex101.com/r/vkijKf/1/.
+  # Break down the regex into readable parts major.minor.patch
```

Comment:
Please add [this link](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) to this comment.  You can add it in place of the regex101 link or include both.

Reply:
@dav3r beat you to it! See here: https://github.com/cisagov/skeleton-generic/pull/176/commits/a77e5e1c9a8752a2072a6a974d4164be116069e9